jobs/src/main/scala/dcos/metronome/utils/glue/MarathonImplicits.scala

```diff
@@ -82,6 +83,13 @@ object MarathonImplicits {
         case _ => None
       }
 
+      val maybeUcr = jobSpec.run.ucr.map { ucrSpec =>
+        Container.MesosDocker(
+          image = ucrSpec.image.id,
+          forcePullImage = ucrSpec.image.forcePull) // TODO: pass privileged once marathon will support it
+      }
+
+      maybeDocker.orElse(maybeUcr)
```

Comment:
does this allow for UCR to be None?   it is possible to have UCR and Docker be None